C-Plus-Plus/dp/LongestPalindromeString.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+In our approach, we use a 2D Boolean array to store the results.
```

Comment:
Add a beginner pitch of the problem too.

Reply:
done